spec/services/appointment_summary_csv_spec.rb

```diff
@@ -49,6 +48,15 @@
           created_at
           email
           notification_id
+          retirement_income_other_state_benefits
```

Comment:
I have no strong feelings either way. It is not adding a great deal of value, but at the same time it does ensure the CSV output is correct.